### PR TITLE
chore(docs): Add doc for the difference between protractor and webdri…

### PR DESCRIPTION
**Pull request by sjelin**
…ver syntax

### CODE REVIEW

File: docs/webdriver-vs-protractor.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+Webdriver Syntax vs Protractor Syntax
+=====================================
+
+In vanilla WebDriver code, you might start your tests with
```
Comment (juliemr) [inline docs/webdriver-vs-protractor.md]: Let's change WebDriver to WebDriverJS and add a link to https://code.google.com/p/selenium/wiki/WebDriverJs

Comment (sjelin) [inline docs/webdriver-vs-protractor.md]: Done.

**Comment by sjelin** [comment]
@juliemr Is this good now?

**Comment by juliemr** [comment]
Yup, looks good.

**Comment by sjelin** [comment]
Merged in at 03d0901

**Pull request closed without merging**